printer_mods/README.md

```diff
@@ -42,6 +42,7 @@ like so:
 ||[Clippable Wago 221 Mounts](./BlueBear/Wago_221_mount)|Mounts for Wago 221 to screw or clip-in on 1515 extrusions | :heavy_check_mark: | :x: | :x: |
 ||[Ziptie Mount](./BlueBear/Ziptie_Mount)|Clippable Ziptie hooks for 1515 extrusions| :heavy_check_mark: | :x: | :x: |
 | btp | [Galileo chain mount (befenybay)](./btp/Galileo_Switchwire_Chain_Mount/README.md) | Modified chain mount position to cater for difference with Galileo Clockwork |:x:|:x:|:x:|:heavy_check_mark:|
+| Conqueror433 | [Voron Parts Logo](./conqueror433/Voron_Parts_Logo/README.md) | Voron Parts Logo for you back panels, 3D Print or Laser Cut | :x: | :x: | :heavy_check_mark: | :x: |
```

Comment:
Change the link to the directory instead of README, that line above yours is incorrect.  So your link should be

    [Voron Parts Logo](./conqueror433/Voron_Parts_Logo)

Reply:
Done